src/test/java/regres/get/users/single/user/GetSingleUserPositiveDataDrivenTest.java

```diff
@@ -18,12 +18,12 @@
 
 @RunWith(SerenityParameterizedRunner.class)
 @WithTags({
-        @WithTag(type= "type", name = "GET")
+        @WithTag(type = "type", name = "GET")
 })
 public class GetSingleUserPositiveDataDrivenTest {
 
     @TestData
-    public static Collection<Object[]> data(){
+    public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][]{
                 {1},
```

Comment:
This is not a good approach, because you do not know for sure these exists. The right approach is to create the users first in a beforeClass method, get the ids and send them as data provider